app/assets/stylesheets/themes/scinote.scss

```diff
@@ -1935,9 +1924,24 @@ th.custom-field .modal-tooltiptext {
   }
 }
 
+.atwho-inserted {
+  align-items: center;
+  display: inline-flex;
+
+  .global-avatar-container {
+    line-height: 30px;
+    margin: 0 5px;
```

Comment:
I think `margin 5px` is to big. If you have another smart annotation, avatar name is closer to the next annotation than to avatar image.
![image](https://user-images.githubusercontent.com/20795228/58092018-5cfa5d00-7bcb-11e9-905d-9b09d6db1b7f.png)